### PR TITLE
 feat: add elm-review spec

### PR DESCRIPTION
**Pull request by wolfadex**
### What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)
Feature: Support for elm-review cli

### What is the new behavior (if this is a feature change)?
Support all of the publicly shown elm-review cli commands as of elm-review version 2.6.1.

### CODE REVIEW

File: src/elm-review.ts [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+        "Hide the details from error reports for a more compact view",
+    },
+    {
+      name: "--ignore-dirs",
+      description:
+        "Ignore the reports of all rules for the specified directories",
+      args: {
+        name: "<dir1,dir2,...>",
+        template: "folders",
+      },
```
Comment (mschrage) [inline src/elm-review.ts:182]: This `arg` object should have `isVariadic: true` set on it

```suggestion
      args: {
        name: "<dir1,dir2,...>",
        template: "folders",
        isVariadic: true,
      },
```

File: src/elm-review.ts [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+    },
+    {
+      name: "--ignore-files",
+      description: "Ignore the reports of all rules for the specified files",
+      args: {
+        name: "<file1,file2,...>",
+        template: "filepaths",
+      },
```
Comment (mschrage) [inline src/elm-review.ts:190]: Should also have `isVariadic: true` enabled.
```suggestion
      args: {
        name: "<file1,file2,...>",
        template: "filepaths",
        isVariadic: true,
      },
```

**Review by mschrage — CHANGES_REQUESTED** [review]
Some minor tweaks but this looks great! Excited to get it merged in. Really appreciate your contribution @wolfadex ✨

File: src/elm-review.ts (hunk 1 above)
Comment (wolfadex) [reply src/elm-review.ts]: Fixed this with the inclusion of the elm-review 2.7.0 changes

File: src/elm-review.ts (hunk 2 above)
Comment (wolfadex) [reply src/elm-review.ts]: Fixed this with the inclusion of the elm-review 2.7.0 changes

**Review by mschrage — APPROVED** [review]

**Comment by mschrage** [comment]
Looks good to me @wolfadex! Merging in now 🤩

### Next steps
💬 **If you are in our [Discord](https://fig.io/community)**: let me know your username so I can give you the 'Contributor' badge!

🐥 **If you have a Twitter**: tweet that `elm-review` is now supported and we'll retweet from the [@fig](https://twitter.com/fig) account :)

🛠 **Are you interested in being a codeowner?** You'll be pinged to review any future PRs for the `elm-review` completion spec.

**Comment by wolfadex** [comment]
@mschrage catching up on stuff after the holidays, I'd be happy to be a codeowner for any of the `elm-*` specs, as AFAIK I'm the only one contributing them right now. I also plan on making PRs for other Elm cli tools in the coming months.

**Pull request merged**